ideascube/mediacenter/forms.py

```diff
@@ -27,3 +30,30 @@ def save(self, commit=True):
         document.save()
         self.save_m2m()
         return document
+
+
+def CreateDocumentForm(path):
+    class NotFileDocumentForm(forms.ModelForm):
+
+        class Meta:
+            model = Document
+            exclude = ['original', 'preview']
+
+        original = forms.FilePathField(path=path)
```

Comment:
So it seems the Django recommanded way of doing this is:

```python
class NotFileDocumentForm(forms.ModelForm):
    class Meta:
        model = Document
        exclude = ['original', 'preview']

    def __init__(self, path, *args, **kwargs):
        super(TicketForm, self).__init__(*args, **kwargs)
        self.fields['original'] = forms.FilePathField(path=path)
        self.fields['preview'] = forms.FilePathField(path=path, required=False)

    def save(self, commit=True):
        ...
```

At least that's what StackOverflow tells me. :stuck_out_tongue: 

It does feel a bit better/simpler than a wrapping function, but it's not as nice since the fields are not defined in their usual place (as class attributes).